ArmPkg/Library/PlatformBootManagerLib/PlatformBm.c

```diff
@@ -1181,3 +1181,56 @@ PlatformBootManagerUnableToBoot (
     EfiBootManagerBoot (&BootManagerMenu);
   }
 }
+
+// MU_CHANGE Starts: Project Mu BDS has this function
+
+/**
+  BDS Entry  - DXE phase complete, BDS Entered.
+*/
+VOID
+EFIAPI
+PlatformBootManagerBdsEntry (
+  VOID
+  )
+{
+  return;
+}
+
+/**
+  HardKeyBoot
+*/
+VOID
+EFIAPI
+PlatformBootManagerPriorityBoot (
+  UINT16  **BootNext

```

Comment:
The next function has `IN` as a modifier on the parameter, for consistency, this one should too.

Reply:
I updated this to IN OUT to reflect the actual usage.